spec/linter-flake8-spec.js

```diff
@@ -12,14 +12,22 @@ describe('The flake8 provider for Linter', () => {
   const lint = require('../lib/main.js').provideLinter().lint;
 
   beforeEach(() => {
+    // This whole beforeEach function is inspired by:
```

Comment:
I'm not sure I want this on _every_ repo, maybe just update https://github.com/AtomLinter/Meta/issues/15 with a link to that discussion?